src/cli/inputHelpers.ts

```diff
@@ -14,8 +14,26 @@
  *  limitations under the License.
  */
 
-declare module 'prompt' {
-  import prompt = require('prompt');
-  // TODO(andreban): Remove usage of any
-  export function get(schema: any): any; // eslint-disable-line @typescript-eslint/no-explicit-any
+import Color = require('color');
```

Comment:
And again with the imports questions, why is this `require`?

Reply:
I don't fully understand the require one, but it's related to the underlying library being written in JavaScript. Using `import * as Color from 'color'`, `import {Color} from 'color'` or `import Color from 'color'` reports the error below:
```
Module '"/Users/andreban/Projects/twa-bootstrap/node_modules/@types/color/index"' can only be default-imported using the 'esModuleInterop' flagts(1259)

index.d.ts(135, 1): This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.
```